debian/control

```diff
@@ -6,7 +6,7 @@ Source: gnome-software
 Section: gnome
 Priority: optional
 Maintainer: Debian GNOME Maintainers <pkg-gnome-maintainers@lists.alioth.debian.org>
-Uploaders: Laurent Bigonville <bigon@debian.org>, Matthias Klumpp <mak@debian.org>
```

Comment:
Is the change to `Uploaders:` intentional?

Reply:
I generate `debian/control` from `debian/control.in` by running `debian/rules clean` which I believe is the intended process. I didn't intentionally change this, it's just what the computer did. Happy to remove this hunk.

Reply:
I removed this by hand. It is generated by `/usr/share/gnome-pkg-tools/1/rules/uploaders.mk`, which derives the list using a gnarly heuristic of `debian/changelog` entries.